.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
+    "editor.rulers": [
+        80
+    ],
+    "editor.wordWrap": "on"
+}
```

Comment:
please do not change anything outside of the student folder - I need you to remove this file please.